Neos.Fusion/Classes/Core/Runtime.php

```diff
@@ -161,21 +145,9 @@ class Runtime
      */
     public function __construct(array $fusionConfiguration, ControllerContext $controllerContext)
     {
-        $this->fusionConfiguration = $fusionConfiguration;
+        $this->runtimeConfiguration = new RuntimeConfiguration($fusionConfiguration);
```

Comment:
Shouldn't this happen in a way that makes it easier to exchange the implementation for testing? I thought that was the general idea (together with performance) for this change. Like a factory for example?

Reply:
Well, right now it was only meant for easier testing of the RuntimeConfiguration itself and better encapsulation of the configuration handling code.

Ideally a `RuntimeConfiguration` value should be passed as a dependency in the constructor, but that would change the signature.